docs/identities/native-browser.mdx

```diff
@@ -0,0 +1,27 @@
+---
+id: native-browser
+title: Browser vs Native Apps
```

Comment:
We use sentence case for document titles and headings ->
`Browser vs. native apps`

Also, note the period in `vs.`. This form is indicated as American English, which is what we use in our writing.

Reply:
```suggestion
title: Browser vs. Native Apps
```